pkg/cleaner/aws/aws_test.go

```diff
@@ -30,6 +33,7 @@ func TestStackShouldBeDeleted(t *testing.T) {
 			stack: &cloudformation.Stack{
 				StackName:    aws.String("host-peer-ci-blblalal"),
 				CreationTime: &now,
+				StackStatus:  &dummyStatus,
```

Comment:
You should use `aws.String()` in those cases in order to create string pointers, therefore you do no need to declare variables before hands.

```go
				StackStatus:  aws.String("FOO"),
```

Reply:
Useful, thanks!